uwblueprint/focus-on-nature
# Camp creation form

## Notion ticket link
[Camp Creation/Editing: Camp Details Form Fields](https://www.notion.so/Task-Board-F22-9e03bf9f7c4343f09a8a5c16dc48a12a?p=37e1bbe6fdbe431e93b71db2c89f2b36&pm=c)



## Implementation description
 - Created camp creation details form
 - Form fields filled validation
 - Image upload
 - Stored all input in state variables as outlined in ticket
 - Added a dummy submit button that acts as a submit so that unfilled form errors are only shown on submission attempt (rather than showing them all the time since the fields are always empty at the start and it doesnt make sense to show all the red messages then)



## Steps to test
1. Fill in the form
2. Press the dummy submit on an empty form
3. Fill in the fields and check if the errors disappear (and reappear if u empty the field again)
4. Upload an image via click and drop or drop
5. Try uploading non image files and files >5MB



## What should reviewers focus on?
Major Changes:
- state variables in the `CampCreation/index.tsx`
- `CampDetails/index.tsx`


## Checklist
